src/main/java/roomescape/exception/GeneralExceptionHandler.java

```diff
@@ -0,0 +1,46 @@
+package roomescape.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GeneralExceptionHandler {
+
+    @ExceptionHandler(DuplicateReservationException.class)
+    public ResponseEntity<String> handleDuplicatedReservation(DuplicateReservationException e) {
+        return ResponseEntity.status(HttpStatus.CONFLICT).body(e.getMessage());
+    }
+
+    @ExceptionHandler(TimeNotFoundException.class)
+    public ResponseEntity<String> handleTimeNotFound(TimeNotFoundException e) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
+    }
+
+    @ExceptionHandler(ThemeNotFoundException.class)
+    public ResponseEntity<String> handleThemeNotFound(ThemeNotFoundException e) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
+    }
+
+    @ExceptionHandler(MemberNotFoundException.class)
+    public ResponseEntity<String> handleMemberNotFound(MemberNotFoundException e) {
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
+    }
+
+    @ExceptionHandler(JwtValidationException.class)
+    public ResponseEntity<String> handleJwtValidationException(JwtValidationException e) {
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
+    }
+
+    @ExceptionHandler(JwtProviderException.class)
+    public ResponseEntity<String> handleJwtProviderException(JwtProviderException e) {
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
+    }
```

Comment:
이렇게 로직이 같은 것들은 
```suggestion
    @ExceptionHandler(MemberNotFoundException.class, JwtValidationException.class, JwtProviderException.class)
    public ResponseEntity<String> handleUnauthorized(Exception e) {
        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
    }

```
같은 느낌으로 통일해보면 어떨까요?
중복을 줄일 수 있을 것 같아요

Reply:
네 좋습니다! 훨씬 깔끔하고 좋은 방식이네요. 
잦은 코드 중복에 유의해야겠어요!